test/support/transform/bom/updateBomFromUpstreamTest.ts

```diff
@@ -99,7 +107,98 @@ const Bom = `<?xml version="1.0" encoding="UTF-8"?>
         <artifactId>hibernate-validator</artifactId>
         <version>\${hibernate-validator.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.apache.tomcat.embed</groupId>
+        <artifactId>tomcat-embed-core</artifactId>
+        <version>\${tomcat.version}</version>
+      </dependency>
     </dependencies>
   </dependencyManagement>
 </project>
 `;
+
+const InitialReadme = `// spring-boot
```

Comment:
Perhaps we could have this as a function that takes a few params and also use for UpdatedReadme?

Reply:
Perhaps, though there are formatting differences between the initial and updated versions (not on purpose but still interesting to know).